### PR TITLE
Fixed device details loading

### PR DESCRIPTION
**Pull request by dev-tim**

### CODE REVIEW

File: public/app/screens/main/sections/device-details/device-details.jsx [hunk 1]
```diff
@@ -45,16 +49,15 @@ var DeviceDetails = React.createClass({
...
-    var device = this.state.deviceList.deviceById[deviceId] || {};
+    var device = this.state.device.device || {};
     var deviceProperties = device.properties || {};
     return (
-
       <div className='cc-device-details grid-16'>
         <div className='cc-settings__header row fixed-height-3'>
           <Link className='cc-device-details__closeButton' to='devices'>
-            <Icon type='close' size='small' />
+            <Icon type='close' size='small'/>
```
Comment (larsblumberg) [inline public/app/screens/main/sections/device-details/device-details.jsx]: I thought we go with a space before `/>`, I'm however fine with both stylings

File: public/app/common/components/device-action-control/device-action-control.jsx [hunk 2]
```diff
@@ -40,10 +40,14 @@ var DeviceActionControl = React.createClass({
   render: function render () {
     var device = this.props.device;
 
-    return React.createElement(
-      this.controlByCapability[device.primaryCapability],
-      this.getControlProps(device)
-    );
+    var result = null;
+    if (device && device.alterEgoComponent) {
```
Comment (DaQuirm) [inline public/app/common/components/device-action-control/device-action-control.jsx]: Could you write a comment on when this condition may not be met?

File: public/app/screens/main/sections/device-details/device-details.jsx [hunk 3]
```diff
@@ -45,16 +49,15 @@ var DeviceDetails = React.createClass({
   },
 
   render: function render () {
-    var deviceId = this.props.params.deviceId;
-    var device = this.state.deviceList.deviceById[deviceId] || {};
+    var device = this.state.device.device || {};
```
Comment (larsblumberg) [inline public/app/screens/main/sections/device-details/device-details.jsx]: Why there's a `device.device` indirection?

**Comment by DaQuirm** [comment]
LGTU (me and @larsblumberg) :smiley_cat:

**Pull request merged**